src/app/content/components/ContentLink.spec.tsx

```diff
@@ -68,35 +68,14 @@ describe('ContentLink', () => {
       expect(event.preventDefault).toHaveBeenCalled();
     });
 
-    it('dispatches navigation action with search if there is a search', async() => {
```

Comment:
@TomWoodward am I right that this is no longer relevant since search params won't automatically be passed on navigation?